docker/images/app/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mhart/alpine-node:16
 
 WORKDIR /home/node
 
-COPY package*.json .npmrc ./
```

Comment:
Not entirely related but if you want to run the app in the container this is needed